docs/source/traitsui_user_manual/factories_basic.rst

```diff
@@ -837,8 +905,8 @@ Example 16: Instance editor with instance selection
         import View, Item, InstanceEditor
 
     class Person ( HasStrictTraits ):
-        name = Str()
-        age = Int()
+        name  = Str
```

Comment:
Urgh. Do we _really_ want to do this?

Reply:
No, that's a regression from previous clean-up.  Not sure why that's happening.